### ISSUE TITLE
BlockEvents errors

### ISSUE THREAD

**Issue by 100petr** [LoneDev6/ItemsAdder#132]
```
[15:19:09 ERROR]: Could not pass event BlockPhysicsEvent to ItemsAdder v2.0.9
java.lang.NullPointerException: null
[15:19:09 ERROR]: Could not pass event BlockBreakEvent to ItemsAdder v2.0.9
java.lang.NullPointerException: null
at dev.lone.itemsadder.d.d.a.b.a.d.a(SourceFile:136) ~[?:?]
at dev.lone.itemsadder.d.d.a.b.a.d.b(SourceFile:105) ~[?:?]
at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.15.2.jar:git-Paper-176]
at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.15.2.jar:git-Paper-176]
at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.15.2.jar:git-Paper-176]
at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.PlayerInteractManager.breakBlock(PlayerInteractManager.java:313) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.PlayerInteractManager.a(PlayerInteractManager.java:272) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.PlayerInteractManager.a(PlayerInteractManager.java:240) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.PlayerConnection.a(PlayerConnection.java:1324) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.PacketPlayInBlockDig.a(SourceFile:40) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.PacketPlayInBlockDig.a(SourceFile:10) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:23) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.TickTask.run(SourceFile:18) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.MinecraftServer.ba(MinecraftServer.java:1058) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.MinecraftServer.executeNext(MinecraftServer.java:1051) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.MinecraftServer.sleepForTick(MinecraftServer.java:1021) ~[patched_1.15.2.jar:git-Paper-176]
at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:944) ~[patched_1.15.2.jar:git-Paper-176]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_241]

**Comment by LoneDev6** [comment]
I think you:
- created custom block
- placed it
- deleted custom block from config
- mined custom block with pickaxe
- got error because it doesn't exist



right?

**Comment by 100petr** [comment]
I didn't do any custom stuff

**Comment by LoneDev6** [comment]
> 
> 
> I didn't do any custom stuff

strange, I will add a check so if unknown block is broken you get clear error in console.

I will update later

**Comment by LoneDev6** [comment]
Should be fixed in next update 2.0.10.
I will publish it today.
Reopen this issue if it's not fixed, thanks!
